bower.json

```diff
@@ -1,32 +1,33 @@
 {
-    "name": "simphony-remote-deps",
```

Comment:
Use four spaces for indentation. I checked the preferred style guide, and most of the opinions are for four spaces, like in python, including Crockford. We don't have to save space, minification will do that (if ever, js files are cached anyway)